npm-scripts.js

```diff
@@ -160,6 +160,7 @@ function buildTypescript(force = false)
 	deleteLib();
 
 	executeCmd('tsc');
+	executeCmd('vite build');
```

Comment:
Why should we run this every time we generate JS code? Why here?

Reply:
Generate js code with vite. tsc is used only for type checking.

Reply:
Is that the `noEmit: true` option in `tsconfig.json`?

Reply:
Yes, it is.